src/bci_build/package/appcontainers.py

```diff
@@ -442,3 +443,36 @@ def _get_nginx_kwargs(os_version: OsVersion):
     )
     for os_version in (OsVersion.TUMBLEWEED,)
 ]
+
+STUNNEL_CONTAINERS = [
```

Comment:
please create a separate stunnel.py toplevel and not add further containers to appcontainers (I apologize for having started this, we definitely shouldn't continue with it)